highfidelity/hifi
# Jump to dev-mobile on start

This PR must affect only debug builds. The idea is jump directly to a domain (hifi://dev-mobile) on start, avoiding other activities that require user interaction (like login menu and domains list)

**Test Plan**
1. Debug builds should open directly in dev-mobile domain
2. Open the app being logged in and logged out
3. Check that release build is not affected

**Manuscript case**
https://highfidelity.manuscript.com/f/cases/19113/Debug-mode-Jump-to-a-domain-on-start
